Makefile

```diff
@@ -21,7 +21,7 @@ PLATFORM ?= linux
 # build variables
 BUILD_TIME_VAR := $(REPO_PATH)/version.BuildTime
 BUILD_VERSION_VAR := $(REPO_PATH)/version.BuildVersion
-LDFLAGS ?= "-X $(SOURCE_DATE_EPOCH:+$(BUILD_TIME_VAR)=$(SOURCE_DATE_EPOCH)) -X $(BUILD_VERSION_VAR)=$(VERSION)"
+LDFLAGS ?= "$(SOURCE_DATE_EPOCH:+-X $(BUILD_TIME_VAR)=$(SOURCE_DATE_EPOCH)) -X $(BUILD_VERSION_VAR)=$(VERSION)"
```

Comment:
I think the `:+` behavior is different in a Makefile (it's a bash-ism), so this probably won't do what you want it to. Hope I didn't lead you astray!